symengine/type_codes.inc

```diff
@@ -53,6 +53,10 @@ SYMENGINE_ENUM(COS, Cos)
 SYMENGINE_ENUM(TAN, Tan)
 SYMENGINE_ENUM(COT, Cot)
 SYMENGINE_ENUM(CSC, Csc)
+// Avoid conflict with SEC macro in solaris's system header
+#ifdef SEC
+#undef SEC
+#endif
 SYMENGINE_ENUM(SEC, Sec)
```

Comment:
I think we should change the first entry to `SYMENGINE_SEC`, `SYMENGINE_CSC`. @certik ?

Reply:
I see. I think we should.